.github/workflows/validations.yml

```
@@ -11,93 +11,174 @@ jobs:
-          cp .env.example .env
-      - name: ffi pristine
-        run: bundle pristine ffi
+          PUPPETEER_SKIP_DOWNLOAD=true pnpm install --frozen-lockfile
+      
+      - name: Install Playwright browsers
+        run: pnpm exec playwright install --with-deps
+      
+      - name: Run Rubocop
+        if: matrix.ci_node_index == 0
+        run: bundle exec rubocop
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)